### DISCUSSION TITLE
Backblaze bookmark does not use Custom URL for Copy/Open URL options

### DISCUSSION THREAD

**Discussion by IAreKyleW00t** [iterate-ch/cyberduck#15403] [Q&A]
**Describe the bug**
Follow-up to #13422 about customizing the "Copy URL" option within CyberDuck. I'm using Backblaze and when accessing it through the CyberDuck UI, both the Copy and Open URL right-click options only allow for the `f005.backblazeb2.com` option, even though both the Path and URL are set in the bookmark correctly.

**To Reproduce**
1. Add a new Bookmark for Backblaze
2. Configure it with it's public URL and corresponding B2 bucket
3. Right-click an item attempt to use Copy/Open URL

**Expected behavior**
It is expected that this would function the same as with S3 as shown below.

**Screenshots**
Bookmark:
![Cyberduck_dFZU2SkhVy](https://github.com/iterate-ch/cyberduck/assets/1072572/a98d6cc2-48f1-4e5d-a03b-16a009032cd7)

Right-Click menu:
![Cyberduck_2YGFS0Vvqb](https://github.com/iterate-ch/cyberduck/assets/1072572/810fafe1-1285-4f9f-aba7-f42e4d413df5)

Working on S3 Bookmark:
![Cyberduck_lNsBdnd3VF](https://github.com/iterate-ch/cyberduck/assets/1072572/07f7f8bd-9485-4e1a-add8-c900f92197dc)

**Desktop (please complete the following information):**
 - OS: Windows 11 x64
 - Version: 8.7.1

**Log Files**

Logs from a fresh start, connecting to the B2 bookmark, then closing the window.
[cyberduck.log](https://github.com/iterate-ch/cyberduck/files/13418747/cyberduck.log)

**Additional context**
N/A

**Comment by tatze96** [comment]
Setting a custom URL through the [WebURL](https://docs.cyberduck.io/cyberduck/bookmarks/#http-url) field is only supported for the protocols FTP(S), SFTP, and WebDAV.

**Reply by IAreKyleW00t** [reply]
Are there any plans to support this for B2/S3? Unfortunately for my use case, being able to easily copy the proper URL is the primary need.
